bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqBinding.java

```diff
@@ -316,6 +316,19 @@ private void computeCommandForItem(Command command, String itemName) {
                 } else if (command.equals(OnOffType.OFF) || command.equals(UpDownType.DOWN)) {
                     myqOnlineData.executeGarageDoorCommand(garageopener.getDeviceId(), 0);
                     beginRapidPoll(true);
+                } else if (command instanceof StringType) {
+                    String stringValue = ((StringType) command).toString();
+                    if (stringValue.equalsIgnoreCase(GarageDoorStatus.OPEN.getLabel())
+                            || stringValue.equalsIgnoreCase(GarageDoorStatus.OPENING.getLabel())) {
+                        myqOnlineData.executeGarageDoorCommand(garageopener.getDeviceId(), 1);
+                        beginRapidPoll(true);
+                    } else if (stringValue.equalsIgnoreCase(GarageDoorStatus.CLOSED.getLabel())
+                            || stringValue.equalsIgnoreCase(GarageDoorStatus.CLOSING.getLabel())) {
+                        myqOnlineData.executeGarageDoorCommand(garageopener.getDeviceId(), 0);
+                        beginRapidPoll(true);
+                    } else {
+                        logger.warn("Unknown string command {}", stringValue);
+                    }
```

Comment:
I agree that accepting string commands is a good thing, but I am concerned that using choices from `GarageDoorStatus` may not be the most appropriate because that enum is about the current state of the garage door, not the commands you can send to the garage door.  Perfectly reasonable string commands could be things like "open", "ON", "1", "close", "OFF", "0", or perhaps a new enum called GarageDoorCommand that produces the 0 or 1 that's needed?